daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -195,6 +195,44 @@ Entity management is only possible when using [Microsoft Entra ID Authentication
 
 > Dapr passes the name of the consumer group to the Event Hub, so this is not supplied in the metadata.
 
+## Receiving custom properties
+
+By default, Dapr does not forward [custom properties](https://learn.microsoft.com/en-us/azure/event-hubs/add-custom-data-event). However, by setting the subscription metadata `requireAllProperties` to `"true"`, you can receive custom properties as HTTP headers.
+
+```yaml
+apiVersion: dapr.io/v2alpha1
+kind: Subscription
+metadata:
+  name: order-pub-sub
+spec:
+  topic: orders
+  routes: 
+    default: /checkout
+  pubsubname: order-pub-sub
+  metadata:
+    requireAllProperties: "true"
```

Comment:
it looks like this needs to be added to the example and the metadata table at the top of the page?

Reply:
The example and table above refer to the pubsub component, while the `requireAllProperties` affects the pubsub subscription. This is similar to the section _Configuring checkpoint frequency_ where the metadata `checkPointFrequencyPerPartition` is set for the subscription.

Do you think it would be beneficial to emphazise more in the text that this affects the subscription and not the component?

Reply:
Ope, I didn't notice that, but what you have is good enough, with `kind: Subscription` - thank you!